packages/legend-data-cube/src/stores/core/__tests__/DataCubeQuerySnapshotBuilder.repl-test.ts

```diff
@@ -103,7 +103,7 @@ const cases: BaseSnapshotAnalysisTestCase[] = [
     'Invalid: Unsupported function composition: select()->filter()',
```

Comment:
this is an invalid combination, why is it allowed now?

Reply:
fixed it, bug on the code end